commerce-price-list-service/src/main/resources/com/liferay/commerce/price/list/internal/upgrade/v3_0_0/dependencies/CommercePriceListUpgradeProcess.sql

```diff
@@ -0,0 +1,8 @@
+UPDATE
+	CommercePriceList
+SET
+	commerceCurrencyCode = CommerceCurrency.code_
+FROM
```

Comment:
`UPDATE...FROM` isn't supported on Oracle I don't think. We should change this to something lamer but more universally-compatible.

Reply:
@ethanbustad , I found something about it like
https://stackoverflow.com/questions/7030699/oracle-sql-update-a-table-with-data-from-another-table
But what do you recommend to use?

Reply:
I can't really find a definitive answer since the ISO documentation for SQL is behind a paywall and all db vendors define their own extensions anyway.... BUT, from what I can tell: there's no way to do this in a single statement that works in all the dbs Liferay supports. `UPDATE...FROM` works for most dbs (but not Oracle), and the Oracle-specific solutions listed in that article don't work for most (any?) other dbs.

So I think we have to do it the stupid way (pseudo-ish code):

    rs = runSQL("select commerceCurrencyId, code_ from CommerceCurrency")

    while (rs.next()) {
        runSQL("update CommercePriceList set commerceCurrencyCode = {0} where commerceCurrencyId = {1}".format(rs.getString("code_"), rs.getString("commerceCurrencyId")));
    }

something like that. Could be batched for better performance -- though there shouldn't be _too_ many currencies anyway.

All that to say, don't think there's a way to do it with one statement that'll work for all the dbs we need it to work for :/